pypeit/reduce.py

```diff
@@ -96,7 +97,7 @@ class Reduce:
     @classmethod
     def get_instance(cls, sciImg, spectrograph, par, caliBrate,
                  objtype, ir_redux=False, find_negative=False, det=1, std_redux=False, show=False,
-                 binning=None, setup=None, basename=None):
+                 binning=None, setup=None, basename=None, manual=None):
```

Comment:
I don't see why we should add a dependency on the manual data container here. Instead can you parse the manual dict outside of this Class and pass in that dictionary instead. That will streamline the functionality between coadd2d and normal reductions and it is also better to have dependencies be standard python objects like dicts rather than pypeit specific data containers whenever possible.

Reply:
I'm not exactly following why the manual_extract.py is a data container, unless you are writing this information to disk, then I would understand. The manual extraction information can easily be stored in a dict, or a list of dictsI, etc. And any routines that parse them using neg=True or False, can remain methods in reduce.py (the only place where this method is required) rather than being a method on the data container. In this way you simplify the coding and remove dependencies on complex objects. Do you disagree?